hw/dv/tools/xcelium/cover.ccf

```diff
@@ -5,6 +5,9 @@
 // Include our common coverage CCF.
 include_ccf ${dv_root}/tools/xcelium/common.ccf
 
+deselect_coverage -betfs -module tb...
+select_coverage -befs -module ${DUT_TOP}...
```

Comment:
@rasmus-madsen can you take a look at this? It only enables code coverage for DUT as we did for VCS.
https://github.com/lowRISC/opentitan/blob/6c8f806de72c0df0d16bdb649e7f015974e9e3fa/hw/dv/tools/vcs/cover.cfg#L8

Reply:
@weicaiyang 
sorry I missed this PR
is there any more for me to do?

Reply:
No worries. LMK if this change doesn't make sense to you.